### PR TITLE
Configure prow/tide for netobserv repos

### PR DESCRIPTION
**Pull request by jotak**
- Adding two repos, goflow2-kube-enricher and network-observability-console-plugin
- No automatic self approval
- Adding either no-qe or qe-approved as necessary labels for merge
- Removed the 4.10 branch (unless I misunderstand what it is, it shouldn't be necessary for this upstream project)

This is a follow-up on this PR [1] which added `netobserv/network-observability-operator`

[1] https://github.com/openshift/release/pull/22876

### CODE REVIEW

**Comment by jotak** [comment]
Note that I also wanted to add this tide config: `rebase_label: no-squash` , in order to be able to override the default merge strategy (squash) per pull-request, as documented here [1], but it doesn't seem possible, I had this error:

```
could not load Prow configuration: [failed to merge in config from /config/netobserv/goflow2-kube-enricher/_prowconfig.yaml: 
only 'branch-protection', 'tide.merge_method' and 'tide.queries' may be set via additional config, all other fields have no merging 
logic yet.
```

[1] https://github.com/kubernetes/test-infra/blob/master/prow/cmd/tide/config.md#general-configuration

**Comment by jotak** [comment]
/approve cancel
(please take a close look at this PR as I'm not super-familiar with prow config)

File: ci-operator/config/netobserv/goflow2-kube-enricher/netobserv-goflow2-kube-enricher-main.yaml [hunk 1]
```diff
@@ -26,14 +25,10 @@ resources:
       memory: 200Mi
 tests:
 - as: ci
-  commands: |
-    export HOME="/tmp"
-    export GOCACHE=/tmp/.cache
-    export XDG_CACHE_HOME=/tmp/.cache
```
Comment (mariomac) [inline ci-operator/config/netobserv/goflow2-kube-enricher/netobserv-goflow2-kube-enricher-main.yaml:32]: I remember some folder permission issues with the make commands without those env vars. Have you verified that they only apply to make test?

**Review by mariomac — APPROVED** [review]
/lgtm
just one minor question.

File: ci-operator/config/netobserv/goflow2-kube-enricher/netobserv-goflow2-kube-enricher-main.yaml (hunk 1 above)
Comment (jotak) [reply ci-operator/config/netobserv/goflow2-kube-enricher/netobserv-goflow2-kube-enricher-main.yaml:32]: Actually, I didn't remove these settings but the github `diff` is screwed up here :-/ 
The file on the left is a removed file for operator/4.10 (which I completely removed) whereas the file on the right is a new one for goflow2-kube-enricher

File: ci-operator/config/netobserv/network-observability-operator/netobserv-network-observability-operator-main.yaml [hunk 2]
```diff
@@ -29,7 +29,7 @@ tests:
     export HOME="/tmp"
```
Comment (jotak) [inline ci-operator/config/netobserv/network-observability-operator/netobserv-network-observability-operator-main.yaml:29]: @mariomac see here I didn't remove these env vars

**Comment by jotak** [comment]
/hold
putting on hold (bot keeps sending messages, is it stuck in an infinite loop?)

File: ci-operator/config/netobserv/network-observability-console-plugin/netobserv-network-observability-console-plugin-main.yaml [hunk 3]
```diff
@@ -25,7 +25,25 @@ resources:
       memory: 200Mi
 tests:
 - as: ci
-  commands: npm run build && npm run lint && npm test
+  commands: |
+    nodejs_version=v14.17.5
```
Comment (mariomac) [inline ci-operator/config/netobserv/network-observability-console-plugin/netobserv-network-observability-console-plugin-main.yaml:29]: can this be parametrized as a higher-level property outside the script?

**Review by mariomac — APPROVED** [review]

File: ci-operator/config/netobserv/network-observability-console-plugin/netobserv-network-observability-console-plugin-main.yaml (hunk 3 above)
Comment (jotak) [reply ci-operator/config/netobserv/network-observability-console-plugin/netobserv-network-observability-console-plugin-main.yaml:29]: to do so, we could create a new make target that installs node, or else have that version in a file in the repo (I've seen this pattern used in other projects). any preference?
(we can do that in a future PR as anyway it requires also changes in the plugin repo)

**Pull request merged**